### PR TITLE
	modified:   WXRenderErrorCode.java  WXSDKInstance.java

### PR DESCRIPTION
**Pull request by alex1226**
增加h5页面检测回调，封装后台weex和h5页面无缝切换机制，为weex页面替换原有h5页面提供热备切换支持。

### CODE REVIEW

File: android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java [hunk 1]
```diff
@@ -490,7 +490,7 @@ public void render(String template, Map<String, Object> options, String jsonInit
    *                 weexversion    Weex version(like 1.0.0)
    *                 appversion     App version(like 1.0.0)
    *                 devid        Device id(like Aqh9z8dRJNBhmS9drLG5BKCmXhecHUXIZoXOctKwFebH)
-   *                 sysversion    Device system version(like 5.4.4、7.0.4, should be used with os)
+   *                 sysversion    Device system version(like 5.4.4�?.0.4, should be used with os)
```
Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java]: ?

File: android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java [hunk 2]
```diff
@@ -467,7 +467,7 @@ public void render(String template, Map<String, Object> options, String jsonInit
...
```
Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java]: what's this?

File: android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java [hunk 3]
```diff
@@ -447,7 +447,7 @@ public void setIWXUserTrackAdapter(IWXUserTrackAdapter adapter) {
...
```
Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java]: ?

**Comment by gurisxie** [comment]
hello @alex1226 ,please read the review result and make sure whether need edit,thanks very much.
HI, dear @asfbot ,my honored that I execute community rule of weex  now. Look forward to communicate with you,^_^

**Comment by sospartan** [comment]
@gurisxie 'asfbot‘ is bot... don't talk to it.

**Comment by gurisxie** [comment]
@sospartan I seen he reply the notice...
image

**Comment by sospartan** [comment]
@gurisxie the bots do can do that with github's API.  Weex's bot do the same thing too.

**Comment by gurisxie** [comment]
@alex1226 please read the review result and make sure whether need edit,thanks very much. You can close this pr [here](https://github.com/apache/incubator-weex/pull/4)

**Pull request closed without merging**